### PR TITLE
Update FindCommand to match characters instead of words

### LINKED ISSUE #58: Update FindCommand

### LINKED ISSUE #114: Error message for file not found in tag is not formatted properly
There should be a spaces at “!”

### PR DESCRIPTION
**Pull request by luo-git**
Resolve #58

Minor bug fix #114

### CODE REVIEW

File: src/main/java/seedu/address/ui/ThemeWindow.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
     private Theme currentTheme = MainWindow.getInstance().getCurrentTheme();
     private Theme selectedTheme = currentTheme;
 
+    // TODO Is ListView supposed to be used as a raw type here?
```
Comment (Ma-Yueran) [inline src/main/java/seedu/address/ui/ThemeWindow.java:23]: I'm not sure, but there were some errors when I tried to specify a type.

**Review by Ma-Yueran — APPROVED** [review]
Looks good to merge!

File: src/main/java/seedu/address/ui/ThemeWindow.java (hunk 1 above)
Comment (Ma-Yueran) [reply src/main/java/seedu/address/ui/ThemeWindow.java:23]: I will try to fix that

**Review by li-s — APPROVED** [review]
LGTM

**Pull request merged**